packages/components/src/index.js

```diff
@@ -51,7 +51,6 @@ export { default as ResponsiveWrapper } from './responsive-wrapper';
 export { default as SandBox } from './sandbox';
 export { default as SelectControl } from './select-control';
 export { default as Spinner } from './spinner';
-export { default as ServerSideRender } from './server-side-render';
```

Comment:
I guess it should be still exposed in `wp.components` for backward compatibility reasons.

Reply:
Maybe we can add this proxy as an inline script to avoid the need to keep the npm dependency forever.